packages/frontend-2/lib/dashboard/helpers/tutorials.ts

```diff
@@ -1,52 +1,80 @@
-import type { TutorialItem } from '~/lib/dashboard/helpers/types'
+import { TutorialProduct, type TutorialItem } from '~/lib/dashboard/helpers/types'
 
-export const tutorials: TutorialItem[] = [
+export const tutorialProducts = Object.freeze(<const>{
+  [TutorialProduct.Grasshopper]: 'Grasshopper',
+  [TutorialProduct.PowerBI]: 'Power BI',
+  [TutorialProduct.Revit]: 'Revit',
+  [TutorialProduct.Rhino]: 'Rhino',
+  [TutorialProduct.AutoCad]: 'AutoCad',
+  [TutorialProduct.Automate]: 'Automate',
+  [TutorialProduct.Blender]: 'Blender',
+  [TutorialProduct.SketchUp]: 'SketchUp',
+  [TutorialProduct.Unreal]: 'Unreal',
+  [TutorialProduct.Unity]: 'Unity'
+})
+
+export const tutorialItems: TutorialItem[] = [
```

Comment:
There has been a new one added last month, maybe we can add it in there directly